aviary/mission/flops_based/phases/energy_phase.py

```diff
@@ -146,6 +148,7 @@ def build_phase(self, aviary_options: AviaryValues = None):
 
         user_options: AviaryValues = self.user_options
 
+        reserve = user_options.get_val('reserve')  # do we need this here?
```

Comment:
You probably don't need it, since the variable isn't used in build_phase. 

In fact, it doesn't even look like it is used at all in the builder or in what it builds.

Reply:
All the tests seem to work without it so I removed it.

Reply:
non-reserve phases don't need to specify `reserve=false`, already implemented :)